test/presenters/supergroups/service_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
```

Comment:
One very minor thing. I'd rename this file to `test/presenters/supergroups_services_test.rb` to match the file name of the services presenter.

Reply:
Sure, I'll do that now.

Reply:
Yes! Typo! I meant to suggest `test/presenters/supergroups/services_test.rb`